### PR TITLE
[PTX-2232] Kill portworx vol driver on node where vol backup is in progress

### PR DESCRIPTION
**Pull request by vinayakshnd**
Jenkins job run with these changes:
https://jenkins.portworx.co/job/tp-backup-vinayak/41/console

Jenkins job without backup test suit running (SetupTeardown,AppTasksDown,AppScaleUpAndDown,VolumeUpdate):
https://jenkins.portworx.co/job/Torpedo/job/tp-dev-test/453/

Jenkins job without backup test suit running 
(VolumeDriverDown,VolumeDriverCrash):
https://jenkins.portworx.co/job/Torpedo/job/tp-dev-test/454/

### CODE REVIEW

File: drivers/scheduler/k8s/specs/pgbench/px-pgbench-storage.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 5Gi
```
Comment (stgleb) [inline drivers/scheduler/k8s/specs/pgbench/px-pgbench-storage.yaml:26]: are these changes related to the subject of the PR? Maybe we can create separate one for that?

Comment (vinayakshnd) [reply drivers/scheduler/k8s/specs/pgbench/px-pgbench-storage.yaml:26]: yes @stgleb , we need this change because pgbench app itself is writing 10GB data [here](https://github.com/portworx/torpedo/blob/master/drivers/scheduler/k8s/specs/pgbench/px-pgbench-app.yaml#L52), so need to increase size of PVC.

File: drivers/backup/backup.go [hunk 2]
```diff
@@ -149,6 +149,10 @@ type Backup interface {
 	// or till timeout is reached. API should poll every `timeBeforeRetry
 	WaitForBackupDeletion(ctx context.Context, backupName string, orgID string,
 		timeout time.Duration, timeBeforeRetry time.Duration) error
+
```
[the remark this answers is not in the document]
Comment (vinayakshnd) [reply drivers/backup/backup.go:152]: Done

**Comment by vinayakshnd** [comment]
Jenkins job run with latest changes:
https://jenkins.portworx.co/job/tp-backup-vinayak/63/console

**Comment by vinayakshnd** [comment]
Latest Jenkins job run with this PR:
https://jenkins.portworx.co/job/tp-backup-vinayak/64/console

File: drivers/backup/portworx/portworx.go [hunk 3]
```diff
@@ -86,6 +87,7 @@ func getKubernetesInstance(cluster *api.ClusterObject) (core.Ops, stork.Ops, err
 	if err != nil {
 		return nil, nil, fmt.Errorf("error initializing stork client instance: %v", err)
 	}
```
Comment (stgleb) [inline drivers/backup/portworx/portworx.go:89]: may you remove redundant blank line?
